Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -584,6 +522,26 @@ namespace EMotionFX
         Update(nullptr, nullptr, PhysicsSetup::ColliderConfigType::Unknown, AzPhysics::ShapeColliderPairList(), nullptr);
     }
 
+    void ColliderContainerWidget::SetFilterString(AZStd::string str)
```

Comment:
nit: str -> filterString

Reply:
update: Also looks like this should be a `string_view` (or be a const ref `AZStd::string`)